doc/examples/dki_tractography.py

```diff
@@ -0,0 +1,316 @@
+                             linewidth=2)
+scene.add(gt_peaks)
+
+"""
+Now we are ready to save and show the figure containing the tensor geometries:
+"""
+
+window.record(scene, out_path='geometry_of_DKI_tensors.png', size=(1200, 1200))
+
+"""
+.. figure:: geometry_of_DKI_tensors.png
+   :align: center
+   ** Geometrical representation of the diffusion tensor (upper panels) and the
+   kurtosis tensor (lower panels) as a function of the crossing intersection
+   angle between two simulated white matter fibers. Fiber ground truth are
+   plotted in red **.
+
+From the figure, we can see that the diffusion tensor does not provide any
+information of crossing fibers. Nevertheless, the kurtosis tensor shows
+to be sensitive to the direction of both crossing fibers for intersection
+angles larger than $30^{o}$. Particulary, in these cases, kurtosis maxima are
+present prependicularly to both fibers (for more information see [Raf2015]_).
+
+By combining the information of both diffusion and kurtosis tensors, we can
+also estimate the DKI based orientation distribution function (DKI-ODF)
+(see [Jen2014]_ and [Raf2015]_). The DKI-ODF function can be computed in Dipy
+from the DiffusionKurtosisFit object:
+"""
+
+ODF = dkifit.dki_odf(sphere)
+
+"""
+We plot below the DKI-ODF in the analogous way done for the directional
+diffusivity and kurtosis values:
+"""
+scene = window.Scene()
+ODF = ODF.reshape((5, 1, 1, len(sphere.vertices)), order='F')
+odf_actor = actor.odf_slicer(ODF, sphere=sphere, scale=0.5)
+scene.add(tensors_actor)
+window.show(scene)
+
+window.record(scene, out_path='DKI_ODF_geometry.png', size=(1200, 1200))
+
+"""
+.. figure:: DKI_ODF_geometry.png
+   :align: center
+   ** Geometrical representation of the DKI based orientation distribution
+   function (ODF) estimate as a function of the crossing intersection angle
+   between two simulated white matter fibers. Fiber ground truth are plotted in
+   red **.
+
+We can see from figure that DKI-ODF peaks are near to the fiber directions
+ground truth. In this way, if the ground truth fibers directions are not known
+(case of real brain data), these can be estimated by finding the DKI-ODF
+maxima. Below we illustrate how this is done the HCP-like brain data.
+
+As mention in :ref:`example_reconst_dki`, for real brain datasets, diffusion
+kurtosis imaging requires some pre-processing to reduce the impact of signal
+artefacts. Here, we'll denoise our data using DIPY's patch2self algorithm (see
+:ref:`example-denoise-patch2self`). Since this procedure takes a little to run,
+we first check if this data was previously denoised and saved from previous
+examples. If this is the case the load the previous processed data.
+"""
+
+if op.exists('denoised_cenir_multib.nii.gz'):
+    img = nib.load('denoised_cenir_multib.nii.gz')
+    den = img.get_fdata()
+else:
+    den = patch2self(data, gtab.bvals)
+    nib.save(nib.Nifti1Image(den, affine), 'denoised_cenir_multib.nii.gz')
```

Comment:
could you use `save_nifti` function (`from dipy.io.image import load_nifti, save_nifti`)